bucket/flameshot.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.9.0",
+    "description": "Powerful yet simple to use screenshot software.",
+    "homepage": "https://flameshot.org/",
+    "license": "GPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/flameshot-org/flameshot/releases/download/v0.9.0/flameshot_portable-0.9.0-win64.zip",
+            "hash": "4e1844c6198f39a1dfbceefc3eb8f373a6d63f7d4dfd97e9a058549744b79ff5",
+            "extract_dir": "flameshot-0.9.0-win64"
+        }
+    },
+    "bin": "bin/flameshot.exe",
+    "shortcuts": [
+        [
+            "bin/flameshot.exe",
```

Comment:
Every manifest in official buckets is using native windows backslashes

Reply:
My bad, that's what happens when switching too much between platforms.